Juriba.Dashworks/Public/New-DwDashboardBarWidget.ps1

```diff
@@ -0,0 +1,64 @@
+Function New-DwDashboardBarWidget {
+    [CmdletBinding(SupportsShouldProcess)]
+    param(
+        [Parameter(Mandatory=$true)]
+        [string]$Instance,
+        [Parameter(Mandatory=$true)]
+        [string]$APIKey,
+        [Parameter(Mandatory = $true)]
+        [int]$DashboardId,
+        [Parameter(Mandatory = $true)]
+        [int]$SectionId,
+        [Parameter(Mandatory = $true)]
+        [string]$Title,
+        [Parameter(Mandatory = $true)]
+        [int]$ListId,
+        [Parameter(Mandatory = $true)]
+        [string]$SplitBy,
+        [Parameter(Mandatory = $false)]
+        [string]$OrderByField = $null,
+        [Parameter(Mandatory = $false)]
+        [bool]$OrderByDescending = $false,
+        [Parameter(Mandatory = $false)]
+        [string]$CategoriseBy = $null
+    )
+        $body        = @{
+            "widgetType"                = "Bar"
+            "displayOrder"              = 0
+            "title"                     = $Title
+            "listId"                    = $ListId
+            "listObjectType"            = "devices"
+            "listObjectTypeId"          = 2
+            "colourSchemeId"            = 3
+            "widgetValuesColours"       = @()
+            "colourTemplateId"          = 1
+            "aggregateFunctionId"       = 1
+            "maximumValues"             = 10
+            "drilldownEnabled"          = $true
+            "maxRows"                   = 0
+            "maxColumns"                = 0
+            "cardTypeIsAggregate"       = $false
+            "splitBy"                   = $SplitBy
+            "aggregateBy"               = ""
+            "orderByDescending"         = if ($OrderByDescending) { $true } else { $false }
+            "orderById"                 = if ($OrderByField) { if ($OrderByField -eq "Count") { 2 } else { 1 } } else { $null }
+            "orderByField"              = $OrderByField
+            "showLegend"                = $true
+            "orientationIsVertical"     = $false
+            "layout"                    = $null
+            "displayDataLabels"         = $false
+            "categoriseBy"              = $CategoriseBy
+            "categorisationViewType"    = "Stacked"
+            "categoriseByMaximumValues" = 10
+        } | ConvertTo-Json
+
+        $contentType = "application / json"
```

Comment:
This should be: "application/json" without the spaces as far as I can tell from:

https://datatracker.ietf.org/doc/html/rfc2045#section-5.1

Reply:
Done.